### PR TITLE
Messaging citizen inbox

### PR DESCRIPTION
**Pull request by Thurfjell**
### Ticket:
- [20905](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/20905)
- [20914](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/20914)
- [20928](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/20928)

#### idea
Rather get this PR in now, and follow up stories would alter component dependencies eg. 
Separate e2e playwright PR when all designs are finalised, all components we expect are completed eg. 
It's not entirely sure under which route this will lie, it is very likely that we throw away the current "home" for citizen messages with this. Remains to be decided by product team.

### Description

Target is the component composition for messaging inbox. 
Using new react design system components via npm

All functionality should work but there are UI elements missing, see caveats.

Areas covered:
- Message lists
- View Message

### Caveats
Tabs and Table components will come from design system package when finished/released. Updating these will be in a separate PR.

We have no standard page layout, thus a main component has been constructed for this scope specifically. We use a flex column as layout generally over the messaging application. 

## Type

- [x] **Dependency upgrade**
- [ ] **Bug fix**
- [x] **New feature**
- [ ] **Dev change**
- [ ] **Additional tests**
- [ ] **Documentation**
- [ ] **Other**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] This change might impact the developer experience of others and should be communicated

### Screenshots:

N/A

### CODE REVIEW

**Review by edge33 — APPROVED** [review]
LGTM

File: apps/messaging/app/[locale]/home/[messageId]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import { getTranslations } from "next-intl/server";
+import { unreadUrl } from "../utils";
+
+type Props = {
+  params: { messageId: string };
+};
+
+export default async function inboxMessage(props: Props) {
+  const messagingSdk = await AuthenticationFactory.getMessagingClient();
+  const message = await messagingSdk.getMessage(props.params.messageId);
```
Comment (SamSalvatico) [inline apps/messaging/app/[locale]/home/[messageId]/page.tsx:16]: I would throw the notFound if you get an error here

Comment (Thurfjell) [reply apps/messaging/app/[locale]/home/[messageId]/page.tsx:16]: Ah, we can do that? I'll look at other examples

Comment (SamSalvatico) [reply apps/messaging/app/[locale]/home/[messageId]/page.tsx:16]: https://github.com/ogcio/life-events/blob/7f6ce93ecbb8db1dbd257fa74361462249b80c46/apps/messaging/app/%5Blocale%5D/admin/send-a-message/%5Baction%5D/page.tsx#L147

**Review by SamSalvatico — APPROVED** [review]

**Review by edge33 — APPROVED** [review]
LGTM

**Review by SamSalvatico — APPROVED** [review]

**Pull request merged**